services/director-v2/src/simcore_service_director_v2/core/settings.py

```diff
@@ -471,10 +471,18 @@ class AppSettings(BaseCustomSettings, MixinLoggingSettings):
     )
 
     # for passing self-signed certificate to spawned services
-    # TODO: fix these variables once the timeout-minutes: 30 is able to start dynamic services
-    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: str = ""
-    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: str = ""
-    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: str = ""
+    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_ID: str = Field(
+        "",
+        description="ID of the docker secret containing the self-signed certificate",
+    )
+    DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME: str = Field(
+        "",
+        description="Name of the docker secret containing the self-signed certificate",
+    )
+    DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME: str = Field(
```

Comment:
I would have kept `DIRECTOR_V2_SELF_SIGNED_SSL_SECRET_NAME` and `DIRECTOR_V2_SELF_SIGNED_SSL_FILENAME` as `Optional[str]` defaulting to `None`.
I find it more clear like these.